blocks/v2-masonry-grid/v2-masonry-grid.js

```diff
@@ -0,0 +1,51 @@
+import { createOptimizedPicture } from '../../scripts/lib-franklin.js';
+import { removeEmptyTags, createElement, isVideoLink } from '../../scripts/scripts.js';
+
+const blockName = 'v2-masonry-grid';
+
+export default function decorate(block) {
+  const ul = createElement('ul', `${blockName}__items`);
+
+  [...block.querySelectorAll(':scope > div > div')].forEach((cell) => {
+    if (cell.childElementCount) {
+      const links = cell.querySelectorAll('a');
+      const videos = [...links].filter((link) => isVideoLink(link));
+      const picture = cell.querySelector('picture');
+
+      // if video or picture available, it's part of the grid
+      if (videos.length > 0 || picture) {
+        const li = createElement('li', [`${blockName}__item`, 'border']);
+
+        if (picture) {
+          const img = picture.lastElementChild;
+          const newPicture = createOptimizedPicture(img.src, img.alt, false);
+          li.prepend(newPicture);
+        }
+
+        if (videos.length > 0) {
+          const video = createElement('video', [`${blockName}__video`], {
```

Comment:
Can add this to script to createVideo with mute, autoplay options, because i see we are using this in many places. For example i see same in Hero PR too